### PR TITLE
Get worker template showing up in VS

### PR DESCRIPTION
**Pull request by Tratcher**
#6817
image

### CODE REVIEW

**Review by davidfowl — APPROVED** [review]
LGTM

File: src/ProjectTemplates/Web.ProjectTemplates/content/Worker-CSharp/.template.config/vs-2017.3.host.json [hunk 1]
```diff
@@ -3,37 +3,19 @@
...
-    "id": "1012"
+    "id": "1026"
   },
   "order": 100,
   "icon": "vs-2017.3/Empty.png",
   "learnMoreLink": "https://go.microsoft.com/fwlink/?linkid=2072778",
   "uiFilters": [ "oneaspnet" ],
-  "supportsDocker": true,
+  "supportsDocker": false,
   "legacyTemplateIdentity": "Microsoft.NetCore.CSharp.EmptyWorker",
```
Comment (phenning) [inline src/ProjectTemplates/Web.ProjectTemplates/content/Worker-CSharp/.template.config/vs-2017.3.host.json]: By the way, you can kill the legacyTemplateIdentity, it was pretty much there to provide consistent telemetry when we swapped moved to the template engine based templates.

**Review by phenning — APPROVED** [review]
Other then my comment about legacyTemplateIdentity, LGTM from an authoring perspective.

**Review by glennc — APPROVED** [review]

**Comment by glennc** [comment]
Merge this now. But I want to look at DockerSupport once a build goes through. Looking at the Dockerfile we have now it looks like it should just work.

**Pull request merged**